hnn_core/tests/test_network.py

```diff
@@ -756,27 +756,55 @@ def test_tonic_biases():
                        target_gids='L2_basket',
                        loc='soma', receptor='ampa', weight=1e-3,
                        delay=1.0, lamtha=3.0)
+
+    tonic_bias_1 = {
+        'name_nonexistent': 1.0
+    }
+
     with pytest.raises(ValueError, match=r'cell_type must be one of .*$'):
-        net.add_tonic_bias(cell_type='name_nonexistent', amplitude=1.0,
-                           t0=0.0, tstop=4.0)
+        net.add_tonic_bias(amplitude=tonic_bias_1, t0=0.0,
+                           tstop=4.0)
+
+    tonic_bias_2 = {
+        'L2_pyramidal': 1.0
```

Comment:
```suggestion
        'L2_pyramidal': 1.0,
        'L5_basket': 0.5
```

Reply:
Better to test that we're able to add multiple tonic biases simultaneously with this new API

Reply:
Done